src/scripts.js

```diff
@@ -15,32 +15,68 @@ const searchRoomsButton = document.getElementById('searchRoomsButton');
 const filterButton = document.getElementById('filterButton');
 const clearFiltersButton = document.getElementById('clearButton');
 const myReservationsButton = document.getElementById('myReservationsNav');
+const loginButton = document.getElementById('loginButton');
+const logoutButton = document.getElementById('logOutNav');
 
 const userReservations = document.getElementById('myReservations');
 const availableRooms = document.getElementById('availableRooms');
 const filteredRooms = document.getElementById('filteredRooms');
 const modalBg = document.getElementById('modalBg');
+const loginPage = document.getElementById('loginPage');
+const loginForm = document.getElementById('loginForm');
 
 searchRoomsButton.addEventListener('click', checkForRooms);
 filterButton.addEventListener('click', filterRooms);
 clearFiltersButton.addEventListener('click', clearRoomOptions);
 myReservationsButton.addEventListener('click', displayUserReservations);
+loginButton.addEventListener('click', validateLogin);
+loginForm.addEventListener('keyup', () => {if(event.key === 'Enter') validateLogin()});
+logoutButton.addEventListener('click', () => location.reload());
 filteredRooms.addEventListener('click', generateModal);
 modalBg.addEventListener('click', collapseModal);
 
-getData();
+function validateLogin() {
```

Comment:
This whole function looks really solid. I tried something similar with the error handling, but couldn't get it to work. I'll follow along with what you did here, see where I went wrong.